images/runtime/dotnetcore/2.2/Dockerfile

```diff
@@ -11,11 +11,12 @@ ENV GIT_COMMIT=${GIT_COMMIT}
 ENV BUILD_NUMBER=${BUILD_NUMBER}
 RUN ./build.sh dotnetcore /opt/startupcmdgen/startupcmdgen
 
-FROM %DOTNETCORE_BASE_IMAGE%
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2.7
 
 RUN apt-get update \
   && apt-get install -y \
      file \
+     lldb \
```

Comment:
For .NET Core versions prior to 3.0, LLDB debugger + SOS (which is part of the runtime folder in this image) is the way to debug the applications.